packages/elements-react/src/locales/index.ts

```diff
@@ -1,11 +1,31 @@
 // Copyright © 2023 Ory Corp
 // SPDX-License-Identifier: Apache-2.0
 
-export { default as de } from "./de.json"
-export { default as en } from "./en.json"
-export { default as es } from "./es.json"
-export { default as fr } from "./fr.json"
-export { default as nl } from "./nl.json"
-export { default as pl } from "./pl.json"
-export { default as pt } from "./pt.json"
-export { default as sv } from "./sv.json"
+import { default as en } from "./en.json"
+import { default as de } from "./de.json"
+import { default as es } from "./es.json"
+import { default as fr } from "./fr.json"
+import { default as nl } from "./nl.json"
+import { default as pl } from "./pl.json"
+import { default as pt } from "./pt.json"
+import { default as sv } from "./sv.json"
+
+// export type TranslationFile = {
+//   [K in keyof typeof en]: string
+// }
```

Comment:
Remove?

Reply:
I left it in there to showcase how we could achieve typesafety.